heartbeat/docs/heartbeat-options.asciidoc

```diff
@@ -123,6 +123,44 @@ value specified for `timeout` is greater than `schedule`, intermediate checks
 will not be executed by the scheduler.
 
 [float]
+[[monitor-fields]]
+==== `fields`
+
+Optional fields that you can specify to add additional information to the
+output. For example, you might add fields that you can use for filtering log
+data. Fields can be scalar values, arrays, dictionaries, or any nested
+combination of these. By default, the fields that you specify here will be
+grouped under a `fields` sub-dictionary in the output document. To store the
+custom fields as top-level fields, set the `fields_under_root` option to true.
+If a duplicate field is declared in the general configuration, then its value
+will be overwritten by the value declared here.
+
+[float]
+[[monitor-fields-under-root]]
+==== `fields_under_root`
+
+If this option is set to true, the custom <<monitor-fields>>
```

Comment:
Change this to say, 
```
the custom <<monitor-fields,`fields`>>
```

I despise hard-coding links, but unfortunately our doc build thinks it's smart and adds text to the link. You end up with this monstrosity when you don't specify link text:

![image](https://user-images.githubusercontent.com/14206422/29339138-b4380684-81cd-11e7-8e80-541f3461ad24.png)

You might not have noticed this before because the doc build only adds the link text if you're pointing to a sub-section. (And the logic for deciding whether something is a sub-section depends not just on the heading level that you use in the asciidoc source, but also the chunk level.)